fvm/crypto/crypto_test.go

```diff
@@ -227,3 +227,53 @@ func TestVerifySignatureFromRuntime(t *testing.T) {
 		}
 	})
 }
+
+func TestValidatePublicKey(t *testing.T) {
+
+	validPublicKey := func(t *testing.T, s runtime.SignatureAlgorithm) []byte {
+		seed := make([]byte, 256)
```

Comment:
Maybe use an existing constant or refactor this magic number to a descriptive constant. Why 256?